include/allegro5/internal/aintern_bitmap.h

```diff
@@ -14,6 +14,11 @@ extern "C" {
 #endif
 
 typedef struct ALLEGRO_BITMAP_INTERFACE ALLEGRO_BITMAP_INTERFACE;
+   
+enum ALLEGRO_BITMAP_INTERNAL_FLAGS
```

Comment:
I'd rather not use flags for this, we could just use a boolean variable in the bitmap.

That said, I'd rather move this into the driver. I.e. add a new vtable entry for `wrap_changed` that is called in `al_set_bitmap_wrap` and store the boolean in `ALLEGRO_BITMAP_EXTRA_OPENGL`, since we only need to do the lazy update there.

Reply:
Yeah. I made that way for simplicity and easy implementation, so no need for extra storage and bigger code changes like adding a new entry in the vtable.